ext/civiimport/ang/crmCiviimport.js

```diff
@@ -294,15 +290,23 @@
               entityConfig = {'soft_credit': $scope.userJob.metadata.entity_configuration[selectedEntity]};
             }
 
-            $scope.userJob.metadata.import_mappings.push({
+            $scope.userJob.template_fields.push({
               name: importRow.selectedField,
               default_value: importRow.defaultValue,
               // At this stage column_number is thrown away but we store it here to have it for when we change that.
-              column_number: index,
+              column_number: index + 1,
```

Comment:
@eileenmcnaughton what does the above comment mean? I'm adding the +1 here to start the count at 1 instead of zero, but... it's thrown away?

Reply:
yeah - I think the underlying code just isn't looking at it